site/tests/shared/sleep.ts

```diff
@@ -0,0 +1,4 @@
+export const sleep = (ms: number) =>
```

Comment:
`import sleep from "sleep-promise"`


It's a dependecy of the repo root, but can make it a dev depency of site too just in case.

Reply:
Thanks! I removed this for now though since I'm not using it